src/Components/Details/Details.js

```diff
@@ -13,7 +14,7 @@ class Details extends Component {
       movieToDisplay: {},
       isLoading: true,
       error: "",
-      trailerToDisplay: {}
+      trailersToDisplay: []
```

Comment:
This makes more sense now as this really should be an array of objects coming from the API request - see PropTypes where this is specified at the bottom of the DetailsDisplay component file